src/jqconsole.coffee

```diff
@@ -694,6 +698,7 @@ class JQConsole
     # (e.g. ">>> "), and the editable text to the left and right of the cursor.
     @$prompt_label = $(EMPTY_SPAN).appendTo @$prompt_current
     @$prompt_left = $(EMPTY_SPAN).appendTo @$prompt_current
+    @$prompt_left.attr 'class', CLASS_USER_TEXT
```

Comment:
It is for identifying the part of the prompt that is changed by the user. This was useful for implementing highlighting (with highlight.js) but maybe there are other way that I am not considering (not js expert here :smile:)